### PR TITLE
TIED-122: Add endpoints for liveness & readiness probes

### PR DESCRIPTION
**Pull request by danipran**
## Description

Add endpoints for liveness & readiness probes.

## Related Issue(s)

**[TIED-122](https://helsinkisolutionoffice.atlassian.net/browse/TIED-122)**

## Motivation and Context

Liveness & readiness probes require endpoints to work.

[TIED-122]: https://helsinkisolutionoffice.atlassian.net/browse/TIED-122?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .docker/django/uwsgi_configuration.ini [hunk 1]
```diff
@@ -33,3 +33,7 @@ worker-reload-mercy = 60  # How long to wait before forcefully killing workers (
 ignore-sigpipe = true
 ignore-write-errors = true
 disable-write-exception = true
+
+# Exclude health and readiness probes from logs
+route = ^/readiness$ donotlog:
+route = ^/healthz$ donotlog:
```
Comment (voneiden) [inline .docker/django/uwsgi_configuration.ini:39]: Nice

File: helerm/urls.py [hunk 2]
```diff
@@ -45,7 +46,21 @@
...
 
+
+#
+# Kubernetes liveness & readiness probes
+#
+def healthz(*_, **__):
+    return HttpResponse(status=200)
+
+
+def readiness(*_, **__):
```
Comment (voneiden) [inline helerm/urls.py:57]: Would it be beneficial to have `@never_cache` for these?

Comment (danipran) [reply helerm/urls.py:57]: Good idea.

File: .docker/django/uwsgi_configuration.ini (hunk 1 above)
Comment (danipran) [reply .docker/django/uwsgi_configuration.ini:39]: Can't take credit for this 😔 copied from here: https://github.com/City-of-Helsinki/kukkuu/blob/master/.prod/uwsgi.ini#L12-L13

File: helerm/urls.py [hunk 3]
```diff
@@ -45,7 +48,25 @@
 router.register(r"record-search", RecordSearchDocumentViewSet, basename="record_search")
 router.register(r"all-search", AllSearchDocumentViewSet, basename="all_search")
 
+
+#
+# Kubernetes liveness & readiness probes
+#
+@require_safe
```
Comment (danipran) [inline helerm/urls.py:55]: Added this for Sonarcloud.

**Review by voneiden — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**